libkernelflinger/android.c

```diff
@@ -1281,15 +1368,33 @@ static EFI_STATUS setup_command_line(
         if (boot_device) {
                 CHAR16 *diskbus = NULL;
 #ifdef AUTO_DISKBUS
+#ifdef USE_SBL
+		int32_t bdf;
+		uint32_t diskbus1, storage_bus_num;
+		diskbus1 = get_diskbus();
```

Comment:
How about rename "diskbus1" to "disk_bus_num", since "diskbus1" is easily to get confused with "diskbus"